esp_encrypted_img/examples/pre_encrypted_ota/conftest.py

```diff
@@ -0,0 +1,9 @@
+import logging
+import os
+
+import pytest
+from _pytest.fixtures import FixtureRequest
+
+@pytest.fixture
+def config(request: FixtureRequest) -> str:
+    return getattr(request, 'param', None) or DEFAULT_SDKCONFIG
```

Comment:
```suggestion
from pytest_embedded.plugin import multi_dut_fixture

@pytest.fixture
@multi_dut_fixture
def build_dir(target: str, config: str) -> str:
    return f'build_{target}_{config}'

@pytest.fixture
@multi_dut_fixture
def config(request: FixtureRequest) -> str:
    return getattr(request, 'param', None) or DEFAULT_SDKCONFIG
```

and run with `pytest --target esp32 .` works

Reply:
Yes, done with these changes